index.js

```diff
@@ -289,6 +289,18 @@ class YouTubePlayer extends EventEmitter {
           loadCb(null, window.YT)
         }
       }
+    } else {
```

Comment:
This whole `else` case should be combined with the previous case to reduce the needless code duplication. I'll fix.